documentation/asciidoc/computers/ai/getting-started.adoc

```diff
@@ -154,8 +151,40 @@ This demo performs 17-point human pose estimation, drawing lines connecting the
 
 [source,console]
 ----
-$ rpicam-hello -t 0 --post-process-file ~/rpicam-apps/assets/hailo_yolov8_pose.json --lores-width 640 --lores-height 640
+$ rpicam-hello -t 0 --post-process-file /usr/share/rpi-camera-assets/hailo_yolov8_pose.json
+----
+
+=== Alternative Package Versions
+
+The AI Kit/HAT+ devices will not function if there is a version mismatch between the Hailo software packages and device drivers. In addition, Hailo's neural network tooling may also require particular a version for the generated model file. At the time of writing, the latest package version is 4.19, but if earlier versions are required, the following steps should be followed:
```

Comment:
Is there a reason why a user might want to explicitly use an older version, rather than just using whatever gets auto-installed by an `apt update`?

Reply:
Some older models won't work with the latest packages.

Reply:
Aha, thanks.